vktrace/vktrace.md

```diff
@@ -17,6 +17,7 @@ Options for the `vktrace` command are:
 | -P&nbsp;&lt;bool&gt;<br>&#x2011;&#x2011;PMB&nbsp;&lt;bool&gt; | Trace  persistently mapped buffers | true |
 | -tr&nbsp;&lt;string&gt;<br>&#x2011;&#x2011;TraceTrigger&nbsp;&lt;string&gt; | Start/stop trim by hotkey or frame range. String arg is one of:<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hotkey-[F1-F12\|TAB\|CONTROL]<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;frames-&lt;startframe&gt;-&lt;endframe&gt;| on |
 | -tpp&nbsp;&lt;bool&gt;<br>&#x2011;&#x2011;TrimPostProcessing&nbsp;&lt;bool&gt; | Enable trim post-processing to make trimmed trace file smaller, see description of VKTRACE_TRIM_POST_PROCESS below | false |
+| -tl&nbsp;&lt;bool&gt;<br>&#x2011;&#x2011;TraceLock&nbsp;&lt;bool&gt; | Enable locking of API calls during trace if TraceLock is set to TRUE, default is FALSEin which it is enabled only when trimming is enabled, see description of VKTRACE_ENABLE_TRACE_LOCK below | false |
```

Comment:
The description text needs to be formatted correctly and probably reworded. Might suggest wording of: "Enable locking of API calls during trace. Default is  TRUE if trimming is enabled, FALSE otherwise. See description of VKTRACE_ENABLE_TRACE_LOCK below".

For the default column, might replace "false" with "See description".

Reply:
Please see commit [d2c224c ](https://github.com/LunarG/VulkanTools/pull/720/commits/d2c224c422d442680b8af020ad8c03a613e2c685). Thanks!